Formula/cfssl.rb

```diff
@@ -18,8 +18,7 @@ class Cfssl < Formula
   depends_on "libtool"
 
   def install
-    ldflags = ["-s", "-w",
-               "-X github.com/cloudflare/cfssl/cli/version.version=#{version}"]
+    ldflags = "-s -w -X github.com/cloudflare/cfssl/cli/version.version=#{version}"
 
     system "go", "build", "-o", "#{bin}/cfssl", "-ldflags", ldflags, "cmd/cfssl/cfssl.go"
     system "go", "build", "-o", "#{bin}/cfssljson", "-ldflags", ldflags, "cmd/cfssljson/cfssljson.go"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: ldflags), "-o", "#{bin}/cfssl", "cmd/cfssl/cfssl.go"
    system "go", "build", *std_go_args(ldflags: ldflags), "-o", "#{bin}/cfssljson", "cmd/cfssljson/cfssljson.go"
```
Since we're making changes to this formula let's use some of the newer syntax for passing `ldflags` to `go`.